### ISSUE TITLE
1.4-Pixel-Perfect-Joey Gaskin

### ISSUE THREAD

**Issue by joeygaskin** [TIY-GVL-FEE-2015-August/assignments#59]
### Tasks

* **Getting Started**
    * [x] Follow the assignment instructions to create a new issue.
    * [x] Follow the assignment instructions to create a new repository.
    * [x] Create and commit `index.html`
    * [x] Follow the assignment instructions to create a new pull request.
    * [x] **Read [this article](http://diveintohtml5.info/semantics.html) about
      semantic HTML.**
* **Make It So**
    * [x] Checkpoint: Basic HTML structure committed!
    * [x] Checkpoint: Basic layout CSS committed!
    * [x] Checkpoint: Pixel perfect page header committed!
    * [x] Checkpoint: Pixel perfect page footer committed!
    * [x] Checkpoint: Pixel perfect column layout committed!
    * [x] Checkpoint: Pixel perfect layout and typography committed!
    * [x] Checkpoint: Links with hover states committed!
    * [x] Mission complete!

**Comment by joeygaskin** [comment]
<b>Hard Mode</b>
<i>Learning Zone: 3.5</i>
<i>Time: 27 hours</i>
Results: Finished! Its not exactly pixel perfect, but this is certainly my best rendering of the image assigned. I learned a lot about the use of 'display: inline-block', and ultimately I'm a lot more comfortable with positioning now than I've ever been. I now have a more complete understanding of the process. Also, I realized I allow myself to get too stuck in the details sometimes. It really keeps me from progressing as quickly as I could. So, I'm learning to keep moving forward, even when I get stuck...
